objsize.py

```diff
@@ -20,8 +20,8 @@
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 """
 import gc
-import sys
 import inspect
+import sys
```

Comment:
Why did you change the order of imports?

Reply:
That probably was my IDE's idea of import optimization - I undid it.